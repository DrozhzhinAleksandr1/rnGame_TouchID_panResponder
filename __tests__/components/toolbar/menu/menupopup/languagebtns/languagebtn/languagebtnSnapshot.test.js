import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import LanguageBtn  from '../../../../../../../app/components/toolbar/menu/menupopup/languagebtns/languagebtn/languagebtn';

it('create snapshot LanguageBtn ', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<LanguageBtn />);
    expect(toolbarSnapshot).toMatchSnapshot();
});
