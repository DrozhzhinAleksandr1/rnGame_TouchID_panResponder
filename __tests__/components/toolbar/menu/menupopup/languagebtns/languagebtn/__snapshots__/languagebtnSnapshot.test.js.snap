// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot LanguageBtn  1`] = `
<TouchableWithoutFeedback>
  <Component
    style={
      Object {
        "height": 24,
        "marginRight": 32,
        "width": 24,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#5A6772",
          "flex": 1,
          "fontFamily": "Roboto-Light",
          "fontSize": 16,
          "lineHeight": 24,
          "textAlign": "center",
        }
      }
    />
  </Component>
</TouchableWithoutFeedback>
`;
